modules/sov-ibc-transfer/src/rpc.rs

```diff
@@ -22,39 +18,50 @@ impl<S> IbcTransfer<S>
 where
     S: Spec,
 {
-    #[rpc_method(name = "escrowedToken")]
+    #[rpc_method(name = "mintedTokenName")]
     pub fn escrowed_token(
         &self,
-        token_denom: String,
+        token_id: TokenId,
         working_set: &mut WorkingSet<S>,
-    ) -> RpcResult<EscrowedTokenResponse> {
-        let token_id =
-            self.escrowed_tokens
-                .get(&token_denom, working_set)
-                .ok_or(ErrorObjectOwned::owned(
-                    jsonrpsee::types::error::UNKNOWN_ERROR_CODE,
-                    format!("No escrowed token found for denom {token_denom}"),
-                    None::<String>,
-                ))?;
+    ) -> RpcResult<MintedTokenResponse> {
+        let token_name = self
+            .minted_token_id_to_name
+            .get(&token_id, working_set)
+            .ok_or(to_jsonrpsee_error(format!(
+                "No IBC-minted token found for ID: '{token_id}'"
+            )))?;
 
-        Ok(EscrowedTokenResponse { token_id })
+        Ok(MintedTokenResponse {
+            token_name,
+            token_id,
+        })
     }
 
-    #[rpc_method(name = "mintedToken")]
+    #[rpc_method(name = "mintedTokenId")]
     pub fn minted_token(
         &self,
-        token_denom: String,
+        token_name: String,
         working_set: &mut WorkingSet<S>,
     ) -> RpcResult<MintedTokenResponse> {
-        let token_id =
-            self.minted_tokens
-                .get(&token_denom, working_set)
-                .ok_or(ErrorObjectOwned::owned(
-                    jsonrpsee::types::error::UNKNOWN_ERROR_CODE,
-                    format!("No minted token found for denom {token_denom}"),
-                    None::<String>,
-                ))?;
+        let token_id = self
+            .minted_token_name_to_id
+            .get(&token_name, working_set)
+            .ok_or(to_jsonrpsee_error(format!(
+                "No IBC-minted token found for denom: '{token_name}'"
```

Comment:
```suggestion
                "No IBC-created token found for denom: '{token_name}'"
```

To be consistent from previous error strings.

Reply:
[e9651e1](https://github.com/informalsystems/sovereign-ibc/pull/133/commits/e9651e12c50544acd44a1d4ce4455c65911aebba)